content/docs/command-reference/data/status.md

```diff
@@ -43,32 +43,38 @@ DVC uncommitted changes:
 (there are other changes not tracked by dvc, use "git status" to see)
 ```
 
-As shown above, the `dvc data status` displays changes in multiple categories:
-
-- _Not in cache_ indicates that the hash for files are recorded in `dvc.lock`
-  and `.dvc` files but the corresponding cache files are missing.
-- _DVC committed changes_ indicates that there are changes that are
-  `dvc-commit`-ed that differs with the last Git commit. There might be more
-  detailed state on how each of those files changed: _added_, _modified_,
-  _deleted_ and _unknown_.
-- _DVC uncommitted changes_ indicates that there are changes in the working
-  directory that are not `dvc commit`-ed yet. Same as _DVC committed changes_,
-  there might be more detailed state on how each of those files changed.
-- _Untracked files_ shows the files that are not being tracked by DVC and Git.
-  This is disabled by default, unless [`--untracked-files`](#--untracked-files)
-  is specified.
-- _DVC unchanged files_ shows the files that are not changed. This is not shown
-  by default, unless [`--unchanged`](#--unchanged) is specified.
-
-By default, `dvc data status` does not show individual changes inside the
-tracked directories, which can be enabled with [`--granular`](#--granular)
-option.
+`dvc data status` displays changes in multiple categories:
+
+- `Not in cache` indicates that there are file records (hashes) in `.dvc` or
```

Comment:
sections names were taken from https://dvc.org/doc/command-reference/status#status for consistency (italic style), again doesn't feel worth creating a PR

could you please highlight any substantial changes in this text please? 

I don't know how to make the process better, but reading 4 paragraphs bc of a few words is just to painful :(

Reply:
On process: There's a reason I reviewed that PR and left feedback (gathering best practices). I didn't intend to make a PR, much less for you to have to review it. Maybe a better process would be to communicate directly to fill any info. gaps (why is this effort happening in the first place) and then decide whether to stop something before getting to this point, or let it run its course.

<details>

<summary>On the style (minor details)</summary>

I used back ticks because this is under a code block where the same strings as shown in monotype, so this made the doc look consistent internally. Our formats and styles should be able to evolve I think, and this was easier to read IMO:

<img width="1302" alt="image" src="https://user-images.githubusercontent.com/1477535/188797789-471f9024-0697-4057-bc4f-ab6e638fbe1d.png">

> Compare to [now](https://dvc.org/doc/command-reference/data/status). 

Also, the format here is already different to https://dvc.org/doc/command-reference/status#local-workspace-status because the latter has sub-lists that we didn't include here (but could).

</details>

I agree that part is minor, but even after I rolled it back (f3fa7c6f9) [the GH diff](https://github.com/iterative/dvc.org/pull/3924/files#diff-fa3d177a2271cc250d345c45de9a7fd83d793ad890ccc4fa7abef1b7a7960921L46-R65) looks almost the same, sorry (the whole block replaced).

> please highlight any substantial changes in this text?

- Improves clarity of explanations in general (and fixes syntax and typos in some cases).
- Mentions scenarios (when each state happens or what to do with it).
- Links to relevant commands.